src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/SignUp.razor

```diff
@@ -20,45 +20,67 @@
         <h1 class="form-title">
             Sign up
         </h1>
-        <div class="form-input-container">
-            <BitLabel>Email</BitLabel>
-            <BitTextField @bind-Value="SignUpModel.Email" Placeholder="Email" />
-            <ValidationMessage For="@(() => SignUpModel.Email)" />
-        </div>
-        <div class="form-input-container">
-            <BitLabel>Password</BitLabel>
-            <BitTextField @bind-Value="SignUpModel.Password"
-                          Type="BitTextFieldType.Password"
-                          CanRevealPassword="true"
-                          Placeholder="Password" />
-            <ValidationMessage For="@(() => SignUpModel.Password)" />
-        </div>
 
-        <div>
-            <BitCheckbox Class="form-checkbox" @bind-Value="SignUpModel.IsAcceptPrivacy">
-                I agree to the <BitLink Href="/privacy" Target="_blank">Bit Privacy Policy</BitLink>
-            </BitCheckbox>
-            <ValidationMessage For="@(() => SignUpModel.IsAcceptPrivacy)" />
-        </div>
+        @if (IsSignedUp is false)
```

Comment:
plz revert this if statement. it's better to check for the `true` value if there is an else block.